src/combat.ts

```diff
@@ -307,76 +309,72 @@ export class Macro extends StrictMacro {
   }
 
   meatStasis(checkPassive: boolean): Macro {
-    // If we don't care about killing the monster don't bother checking passave damage
-    if (!checkPassive) {
-      return this.trySkill($skill`Pocket Crumbs`)
-        .trySkill($skill`Extract`)
-        .externalIf(
-          myFamiliar() === $familiar`Hobo Monkey`,
-          Macro.while_(
-            `!match "shoulder, and hands you some Meat." && !pastround  5 && !hppercentbelow 25`,
-            Macro.item($item`seal tooth`)
-          )
-        )
-        .externalIf(
-          [
-            $familiar`Cocoabo`,
-            $familiar`Feather Boa Constrictor`,
-            $familiar`Ninja Pirate Zombie Robot`,
-            $familiar`Stocking Mimic`,
-          ].includes(myFamiliar()),
-          Macro.while_("!pastround 10 && !hppercentbelow 25", Macro.item($item`seal tooth`))
-        )
-        .externalIf(
-          haveEquipped($item`Buddy Bjorn`) || haveEquipped($item`Crown of Thrones`),
-          Macro.while_("!pastround 3 && !hppercentbelow 25", Macro.item($item`seal tooth`))
-        )
-        .tryItem($item`porquoise-handled sixgun`);
+    // Determine stasis item to use
+    // Garbo already gets a seal tooth at the start of the day, so that's around always
+    let stasisItem = $item`seal tooth`;
+    if (retrieveItem($item`facsimile dictionary`)) {
+      // The dictionaries are nicer though as they don't hurt the monster
+      stasisItem = $item`facsimile dictionary`;
+    } else if (retrieveItem($item`dictionary`)) {
+      stasisItem = $item`dictionary`;
+    };
+
+    // Construct the monster HP component of the stasis condition
+    // Start by evaluating the passive damage
+    const passiveDamage =
+      maxPassiveDamage() + 5;
+    // If we don't care about killing the monster don't bother checking passive damage
+    let monsterHpCheck = '';
```

Comment:
make this a const, and use ternary notation to make it sometimes an empty string. Alternately, use ternary to sometimes not make it part of the BALLS condition it gets used in later.